src/blogs/storage.md

```diff
@@ -0,0 +1,73 @@
+---
+
+title: Monitoring DreamHack - the World's Largest Digital Festival
+author: Samsung
+author_info: Software Engineer at MayaData, working on Cloud Native Tech
+tags: tutorials
+image: img2.png
+avatar: captain.png
+date: 22-03-2021
+content: Christian Samsung (DreamHack Network Team)
```

Comment:
Same as above

Reply:
Should be removed, kept for testing purose.